lib/app_web/templates/item/edit.html.heex

```diff
@@ -0,0 +1,22 @@
+<.container>
+<.h2 class="text-center mt-3">Edit Item's lists</.h2>
```

Comment:
`Edit Item's lists` what is the intent here? Is it to add an `item` to a `list`?

Reply:
> Is it to add an item to a list?

Yes I'm displaying the possible lists as checkboxes for an item to be added to.

For each item I display the `edit list` link:
![image](https://user-images.githubusercontent.com/6057298/195024915-d21b34e8-dab8-4801-86b1-36ba18939e87.png)

Then the user can select which list the item can be added/removed to:
![image](https://user-images.githubusercontent.com/6057298/195025118-ed6989ee-36ae-4579-b725-9bffc278e934.png)

This is only one of the possible workflow that I imagined and I agree it might not be the one you had in mind so we can remove this